src/jit/ByteCodeParser.cpp

```diff
@@ -1932,7 +1932,7 @@ void Module::jitCompile(ModuleFunction** functions, size_t functionsLength, int
     } else {
         do {
             if ((*functions)->jitFunction() == nullptr) {
-                if (verboseLevel >= 1) {
+                if (JITFlags & JITFlagValue::JITVerbose || JITFlags & JITFlagValue::JITVerboseColor) {
```

Comment:
What is the difference between `JITVerbose` and `JITVerboseColor` option?

Reply:
And `JITVerboseColor` is used only here, is this option necessary?

Reply:
With JITVerboseColor it logs colored output. Before that it was handled with verboseLevel (1 was JITVerbose , but flags for JIT (for handle enable JIT, enable PerfDump, setting up logging) is better than 3 parameters. But yes, here JITVerbose is enough.

Reply:
When you want to put the output in a file (e.g. for testing), it is better if it is not full with coloring escape sequences, which is hard to read. On the screen coloring allows easier processing the data. So JITVerboseColor enables coloring the verbose data.

Reply:
What about merging `JITVerboseColor` into `JITVerbose` option?
Are these options needed to be set separately?

Reply:
I am not sure I understand your suggestion. Shall we create only one flag or does verbose color should automatically set verbose as well?

Reply:
The current status looks fine, no need to update more :)